src/pnpq/devices/switch_stub.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from pnpq.errors import (
+    DeviceDisconnectedError,
+)
+
+class Switch:
+    """Stub Switch Device Class"""
+
+    def __init__(self):
```

Comment:
I missed this in the earlier PR, but, here, I started wondering, "what should the return type of `__init__` be?"

And it turns out that [the type annotations PEP explicitly says that the return type of `__init__` should be `None`](https://peps.python.org/pep-0484/#the-meaning-of-annotations).

@TomShen1234 I missed a couple of type annotations on the waveplate PR. If you or @master-of-ppap could add them, it would be appreciated.

```suggestion
    def __init__(self) -> None:
```